frontend/pages/_app.tsx

```diff
@@ -1,6 +1,7 @@
 import { SessionProvider } from "next-auth/react";
 import type { AppProps } from "next/app";
 import "@frontend/styles/globals.scss";
+import "@frontend/styles/doctor.scss";
```

Comment:
should either use `module.scss` in your page/component only or use chakra-ui style props, because the scss globaly importanted and is uselessly rendered if not on doctor's page